dependencies/boost/CMakeLists.txt

```diff
@@ -67,6 +67,7 @@ set( ${PROJECT_NAME}_INSTALL_DIR ${INSTALL_DIR} )
 add_library(${PROJECT_NAME} INTERFACE)
 target_include_directories(${PROJECT_NAME} INTERFACE
   $<BUILD_INTERFACE:${${PROJECT_NAME}_INSTALL_DIR}/include>
+  $<BUILD_INTERFACE:${${PROJECT_NAME}_INSTALL_DIR}/src>
```

Comment:
It seems odd that anything should be dependent on the src directory. My understanding was that if it's in src then it gets built into a lib, otherwise it gets copied to include for header-only libraries.